source/payment_flow/index.html.md.erb

```diff
@@ -318,30 +318,23 @@ The `state` array within the JSON lets you know the outcome of the payment:
 See the [__Making payments__](/making_payments/#making-payments) section for
 more information about how you can integrate your service with GOV.UK Pay.
 
-## Resume a payment
+### Resuming an incomplete payment
 
-If your user starts a payment but does not complete it, they can resume that
-incomplete payment. An example of this could be if they use an internet
-browser’s ‘back’ button during a payment, and then go forward by selecting
-links on your website.
+You can send your user back to GOV.UK Pay if they arrive at your `return_url` before they've finished their payment. For example, your user selected their web browser’s **Back** button while making a payment, then selected **Forward**.
```

Comment:
I don’t think we should specifically mention the `return_url` as it’s unlikely the user will end up there; it’s more likely they’ll end up at an earlier page.

For example, imagine the flow of pages was like this:

https://www.example.service.gov.uk/apply/confirm-details (where the payment gets created) → GOV.UK Pay (various URLs) → https://www.example.service.gov.uk/apply/payment-complete (return URL)

Unless the user knows the return URL already, they’re unlikely to end up at https://www.example.service.gov.uk/apply/payment-complete without completing their payment. However, if they press the back button, they might end up at https://www.example.service.gov.uk/apply/confirm-details. At this point, the service could go, “Hang on, I’ve already created a payment for this user!” Then the service could check the status of the payment and offer to let the user resume if it’s unfinished (or continue with the rest of the journey if it’s finished).

Also, I think the change from “if they use an internet browser’s ‘back’ button during a payment, and then go forward by selecting links on your website” to “You can send your user back to GOV.UK Pay if they arrive at your `return_url` before they've finished their payment. For example, your user selected their web browser’s **Back** button while making a payment, then selected **Forward**” has lost some nuance. I don’t think it was trying to say that the user presses the back button and then the forward button in their browser. I think it was trying to say that if the user presses the back button in their browser that the service _could, by supporting the resume payment feature, show a link allowing the user to resume (go “forward” into) their payment.

Reply:
Thanks Alex - makes sense. As discussed, I've scrunched this down to one sentence in https://github.com/alphagov/pay-tech-docs/pull/332/commits/7924b40b4e4d63cc7d3dd4078cc21be51d6e28b6 that I think does the job?